model/database/tests/test_access.py

```diff
@@ -79,6 +79,40 @@ def test_get_experiment(self):
         self.assertIsNotNone(actual)
         self.assertEqual(123, actual.reference_number)
 
+    @patch('model.database.access.save_record')
+    def test_get_experiment_create(self, mock_save):
+        """
+        Test: An Experiment record is created
+        When: get_experiment is called with create option True
+        """
+        database = access.start_database()
+        actual = access.get_experiment(rb_number=9999999, create=True)
+        self.assertIsNotNone(actual)
```

Comment:
Is this test necessary to check if not None? 
Would this not be highlighted in the assert equality checks.